### ISSUE TITLE
Please Merge into the diffuser repo

### ISSUE THREAD

**Issue by ghost** [Newbeeer/diffusion_restart_sampling#5]
Hi,
I am a fan of the restart scheduler and would love for it to be incorporated into Diffusers! I see you have that listed as a "to do" on your project page so I just wanted to let you know there is still interest and we would love to see this happen.

Thank you!

**Comment by gohan2091** [comment]
This would be great!

**Comment by Newbeeer** [comment]
Thanks for the reminder. I will do that we I have time :)

**Comment by MysticDaedra** [comment]
Another vote for diffusers integration/compatibility. Would love to use this sampler with the newest and greatest optimizations.

**Comment by Aptronymist** [comment]
Just so you know, SD.Next would be happy to include Restart as a sampler, but this still needs to happen first.
Perhaps we could assist a little? I'm betting vlad could find a little time to help out with it if need be, so don't hesitate to drop by our discord (you can just @ me) or hit up our thread on the subject.
